.github/workflows/build-and-upload-archives-on-schedule.yml

```diff
@@ -2,7 +2,7 @@ name: Venice Publication Pipeline
 
 on:
   schedule:
-    - cron: "30 0 * * 3"
+    - cron: "30 20 * * 3"
   workflow_dispatch:
 
 jobs:
```

Comment:
If the deploy keys work, we'll need to pass them via env vars. Check this for an example in another action: https://github.com/linkedin/venice/blob/main/.github/workflows/build-and-upload-archives-upon-creating-tags.yml#L38